backup.gemspec

```diff
@@ -38,13 +38,14 @@ Gem::Specification.new do |gem|
   gem.add_dependency "net-sftp", "2.1.2"
   gem.add_dependency "mail", "~> 2.6", ">= 2.6.6"
   gem.add_dependency "pagerduty", "2.0.0"
-  gem.add_dependency "twitter", "~> 5.5"
+  gem.add_dependency "twitter", "~> 5.16.0"
   gem.add_dependency "hipchat", "1.0.1"
   gem.add_dependency "flowdock", "0.4.0"
   gem.add_dependency "dogapi", "1.11.0"
   gem.add_dependency "aws-sdk", "~> 2"
   gem.add_dependency "qiniu", "~> 6.5"
   gem.add_dependency "nokogiri", "~> 1.8", ">= 1.8.1"
+  gem.add_dependency "http", "~> 1.0"
```

Comment:
Not sure if this addition is necessary? The twitter gem already sets it. Can we remove it? v5 no longer lists all its dependency's dependencies.